hydra_agent/graphutils.py

```diff
@@ -0,0 +1,120 @@
+from redisgraph import Node, Edge, Graph
+from typing import Union, Optional
+from redis.exceptions import ResponseError
+
+
+class GraphUtils:
+
+    def __init__(self, redis_proxy, graph_name='apidoc'):
```

Comment:
Do we really want the graph name to be `apidoc`? This might create confusion

Reply:
> I suggest to change the name to apigraph or we can get confused

Yeah, Lorenzo had the same query, I was just following the name Sandeep used but I agree on changing it. I'll make the modification everywhere where it's necessary soon.